modules/tensor_mechanics/include/materials/PowerLawCreepStressUpdate.h

```diff
@@ -20,59 +19,51 @@ InputParameters validParams<PowerLawCreepStressUpdate>();
 
 /**
  * This class uses the Discrete material in a radial return isotropic creep
```

Comment:
Could you remove all references to a Discrete material? At one point, we had a special Material base class called DiscreteMaterial that accomplished the same thing as what we now do by setting `update=false`. You could call it an Update material or a compute=false material or something.